addon/mixins/adapter-fetch.js

```diff
@@ -4,74 +4,14 @@ import { assign, merge } from '@ember/polyfills'
 import RSVP from 'rsvp';
 import fetch from 'fetch';
 
+import { serializeQueryParams } from '../params';
+
+export { serializeQueryParams } from '../params';
```

Comment:
backwards-compatibility / not creating a breaking change :)

Reply:
I think we can prob do a major version for this change. It was only exporting that function to make testing easier, so to keep it clean it would be nice not to re-export.

But please hold off for now, let me get consensus on publishing a major version.

Reply:
coolio